src/dysh/spectra/spectrum.py

```diff
@@ -1400,3 +1509,66 @@ def spectrum_reader_gbtidl(fileobj, **kwargs):
     # registry.register_writer("ipac", Spectrum, ascii_spectrum_reader_ipac)
     # registry.register_writer("votable", Spectrum, spectrum_reader_votable)
     # registry.register_writer("mrt", Spectrum, spectrum_reader_mrt)
+
+
+def average_spectra(spectra, equal_weights=False):
+    """
+    Average `spectra`.
+
+    Parameters
```

Comment:
Future enhancement: align=T/F to align all spectra to the first in the list.

Reply:
Created issue #402